source/lib/JMSLab/tests/output_macros.tex

```diff
@@ -0,0 +1,2 @@
+\newcommand{\Epsilon}{-1.19}
```

Comment:
@rcalvo12, is this test used somewhere?

Reply:
@veli-m-andirin From looking into it, it seems like this file was a temp test file produced by `test_autofill.py` but is supposed to be cleaned up. My best guess is that somewhere along the way as we were working on testing and this script was run but was interrupted and so the file was produced but not cleaned up. I removed the file in https://github.com/JMSLab/Template/pull/56/commits/3710921502e0e46307ca1104fe5653a92ef3effe and had no problem re-running scons or the testing. Thanks for the catch!